po/ru.po

```diff
@@ -6,7 +6,7 @@ msgstr ""
 "Project-Id-Version: newsbeuter 1.3\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2016-10-25 14:43+0300\n"
-"PO-Revision-Date: 2008-11-18 17:38+0300\n"
+"PO-Revision-Date: 2016-11-19 02:27+0300\n"
 "Last-Translator: Sergey Dryabzhinsky <sergey.dryabzhinsky@gmail.com>\n"
 "Language-Team: ru <sergey.dryabzhinsky@gmail.com>\n"
```

Comment:
Please update these two lines (unless it's your name; couldn't deduce from your profile.)